docs/publishers/events.rst

```diff
@@ -0,0 +1,48 @@
+Custom DOM events
+=================
+
+Hypothesis client emits a few custom DOM events to which your app can react.
+
+.. option:: hypothesis:layoutchange
+
+   This event is emitted when the sidebar layout changes. Changes to the layout occur on load, when the sidebar is
```

Comment:
```suggestion
   This event is emitted when the sidebar layout changes. eg. when it is opened or closed.
```

I don't think we should list all the events that can trigger this, since they may change in future.